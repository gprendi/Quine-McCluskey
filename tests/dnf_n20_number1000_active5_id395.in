d 20
00000000000000010000
00001000000000000000
11000000000000000000
01001000000000010000
00000000000000000000
10001000000000000000
11000001000000010000
00000001000000010000
01000000000000000000
10001001000000000000
11001001000000000000
01001000000000000000
01001001000000000000
10001000000000010000
11001001000000010000
10000001000000010000
01000000000000010000
01000001000000000000
10000000000000000000
10000001000000000000
00001001000000010000
00000001000000000000
11001000000000000000
01001001000000010000
10000000000000010000
11000001000000000000
10001001000000010000
11000000000000010000
00001000000000010000
11001000000000010000
01000001000000010000
00001001000000000000
