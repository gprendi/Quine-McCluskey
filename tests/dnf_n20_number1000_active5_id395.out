p dnf 20 0
-3 -4 -6 -7 -9 -10 -11 -12 -13 -14 -15 -17 -18 -19 -20 0
